src/main/java/net/sf/jabref/logic/formatter/casechanger/Title.java

```diff
@@ -16,6 +17,11 @@ public Title(String title) {
         this.words.addAll(new TitleParser().parse(title));
     }
 
+    public List<Word> getWordsReadOnly() {
```

Comment:
Makes sense. I'll add immutable `getWords` as default and add a comment. On the other hand it is quite convenient that `Word` mutable. Somehow the whole `Title` class depends on it as the `toString()` method really wouldn't make sense if you cannot change any `Word` in `Title`. You would just get the same string that you entered and there is no way to convert a stream to a new `Title` at the moment.